### DISCUSSION TITLE
Get status of solution for Vehicle Routing Problem

### DISCUSSION THREAD

**Discussion by rahulj2409** [google/or-tools#2659] [Routing (and legacy CP) questions] [answered]
I am currently following the online example for the Vehicle Routing Problem and I am trying to get the status of the solution for further debugging. Online it says "solver.status()" would work but where is that in the VRP instance? I thought it would be like "routing.solver().status()" but I get the error that no such method exist.

**Comment by Mizux** [comment, marked as answer]
https://github.com/google/or-tools/blob/b37d9c786b69128f3505f15beca09e89bf078a89/ortools/constraint_solver/routing.h#L208-L222

https://github.com/google/or-tools/blob/b37d9c786b69128f3505f15beca09e89bf078a89/ortools/constraint_solver/routing.h#L1046-L1047

**Reply by rahulj2409** [reply]
What method do I have to write to get the status? "routing.status()"?

**Reply by apoorv-lyric** [reply]
This is not a proper answer of the question. Please give example code that prints the solution status, not the RoutingModel code.
